src/v/kafka/server/group.cc

```diff
@@ -2837,13 +2837,20 @@ ss::future<cluster::abort_group_tx_reply> group::do_abort(
         co_return make_abort_tx_reply(cluster::tx::errc::stale);
     }
     auto it = _producers.find(pid.get_id());
-    if (it == _producers.end()) {
+    if (it == _producers.end() || it->second.transaction == nullptr) {
+        // It could be a replay request from the coordinator to roll back
```

Comment:
should this comment be saying `abort` request not `reply` ?

Reply:
By 'replay' I mean a retried request (it could be a roll back and roll forward)